### DISCUSSION TITLE
Can i hide the message of ApolloServerPluginUsageReporting "apollo schema reporting starting!..."

### DISCUSSION THREAD

**Discussion by SuperWallaby** [apollographql/apollo-server#6220] [Q&A]
Can i hide the message of ApolloServerPluginUsageReporting "apollo schema reporting starting!..."

<img width="373" alt="image" src="https://user-images.githubusercontent.com/46065365/158518916-c98880e9-1be5-46bd-a8f1-c4e03e454be3.png">

`ApolloServerPluginUsageReporting<Context>({...})`

**Comment by glasser** [comment]
Yes, by specifying a custom `logger` to `new ApolloServer`.
